include/openssl/bn.h

```diff
@@ -136,14 +136,7 @@ int BN_is_word(const BIGNUM *a, const BN_ULONG w);
 int BN_is_odd(const BIGNUM *a);
 
 # define BN_one(a)       (BN_set_word((a),1))
-
-void BN_zero_ex(BIGNUM *a);
```

Comment:
@richsalz said:
> No, just from the header file.  It was never documented and it was never intended to be used.

You're removing a declaration from a *public* header file, thus you're removing public API.  Note that this affects `util/libcrypto.num`, you'll see that a `make update` will mark that function `NOEXIST` (as a matter of fact, Travis fails because you haven't done `make update`), thus making it unaccessible to anyone who upgrades the shared library from OpenSSL 1.1.0 (for example anyone who uses `BN_zero` as defined in this same header file), which is a regression.  So not only are you making counter policy API changes, but you're also making counter policy ABI changes.

This is not ok.